src/core/db/manager.py

```diff
@@ -526,6 +931,14 @@ def create_user(username, password, create_db=True):
         return res
 
     @staticmethod
+    def create_user_database(username):
+        """Creates the database and data folder for a user."""
+        with _superuser_connection() as conn:
+            res = conn.create_user_database(username=username)
+            DataHubManager.create_user_data_folder(username)
```

Comment:
It was like that when I found it, so I left it alone: https://github.com/datahuborg/datahub/commit/e9ce3cdd42d0117f904fab00f7f9ed0771d78439#diff-1d4a1f6a64cd4d44cf699a01ec214f27R624. I just removed it in https://github.com/datahuborg/datahub/commit/d226b05f9f783ed817d3abfe61ee3f6d14a852a9.